### ISSUE TITLE
Thunderbird Theme Links Broken

### ISSUE THREAD

**Issue by mltco78dhs** [mozilla/addons-server#9314]
Don't feel like wasting more time on this than is absolutely necessary but every single theme I attempted to install through the in-app choose-a-theme feature of thunderbird resulted in a message that said I'd mistyped the address or clicked an out-of-date link.

I have vision issues that are made far worse by white screens and it's just his side of physically painful to view them. I had a very dark theme that has apparently been made obsolete by the latest update and I need it back.

The built in 'Dark' theme doesn't do a thing for the main part of the page, only the stuff round the edges.

Please unfuck whatever you fucked up and let me run my prior themes again or fix the in-app links and allow me to get to the themes you do offer without error.

**Comment by muffinresearch** [comment]
Add-ons for Thunderbird have moved to https://addons.thunderbird.net. As part of that move, the bug tracker for the add-ons site for Thunderbird has moved here [1].

See also https://github.com/thundernest/addons-server/issues/19 which might have some relevant information.

[1] https://github.com/thundernest/addons-server/issues/new

**Comment by mltco78dhs** [comment]
G-d I freaking hate Github.

You guys seem determined to do anything except fix the problem.

It's all about the epeen - who knows the secret phrase of the day - who's fluent in the current geek-speak.

The freaking program directed me here. Let's start with fixing that crap first. Then concentrate on the issue, not where it was reported.

At a minimum transfer the issue, don't close it. The issue isn't remotely resolved.

Shoving it off your desk onto the floor isn't "resolving it" at all.

To be clear, I never chose a website to go to. I clicked a link within Thunderbird to obtain new themes. When that failed, I never chose to come here to report it. I clicked a link to so so.

If the links under the buttons or other controls in the application are misdirecting users to the wrong places, I think something more than "NGGGGGGAAAAAA, Wrong!" is in order.

We're not all willing go digging into the esoterica of why an application fails.

The nature of a bug report is that the user reports it. 

There isn't any reasonable model where the user is sent on a link chase across the internet hunting down the RIGHT place or being sent off to do research.

If you folks don't want to do this, don't. But don't close tickets that haven't, in fact, been resolved.
